generators/service-push/templates/swift/dependencies.txt

```diff
@@ -1 +1 @@
-.package(url: "https://github.com/ibm-bluemix-mobile-services/bms-pushnotifications-serversdk-swift.git", .upToNextMinor(from: "0.6.0")),
+.package(url: "https://github.com/ibm-bluemix-mobile-services/bms-pushnotifications-serversdk-swift.git", .exact("0.6.2")),
```

Comment:
@tunniclm  Why the exact?

Reply:
This repo has made some breaking changes in patch releases, fortunately they didn't cause a problem. To be safe and prevent future breakage here I went with exact for the version.